spinalcordtoolbox/moco.py

```diff
@@ -189,10 +192,33 @@ def moco_wrapper(param):
     # Start timer
     start_time = time.time()
 
-    printv('\nInput parameters:', param.verbose)
-    printv('  Input file ............ ' + param.fname_data, param.verbose)
-    printv('  Group size ............ {}'.format(param.group_size), param.verbose)
-
+    printv(f"""
+Input parameters:
+----------------------------------------------------
+Input file:            {param.fname_data}
+Output folder:         {os.path.abspath(param.path_out)}
+Mask:                  {param.fname_mask if param.fname_mask != '' else 'None'}
+Reference image:       {param.fname_ref if param.fname_ref != '' else 'None'}
+bvals (dmri only):     {param.fname_bvals}
+bvecs (dmri only):     {param.fname_bvecs}
+    """.format(param=param), param.verbose)
```

Comment:
Nit: I don't think the `.format` is necessary anymore since you are using the f-string feature?